main.tf

```diff
@@ -186,8 +193,10 @@ resource "aws_lb_listener" "this" {
   count = "${length(var.services) > 0 ? length(var.services) : 0}"
 
   load_balancer_arn = "${element(aws_lb.this.*.arn, count.index)}"
-  port              = "80"
-  protocol          = "HTTP"
+  port              = "${lookup(var.services[element(keys(var.services), count.index)], "acm_certificate_arn", "") != "" ? 443 : 80}"
+  protocol          = "${lookup(var.services[element(keys(var.services), count.index)], "acm_certificate_arn", "") != "" ? "HTTPS" : "HTTP"}"
+  ssl_policy        = "${lookup(var.services[element(keys(var.services), count.index)], "acm_certificate_arn", "") != "" ? "ELBSecurityPolicy-2016-08" : ""}"
```

Comment:
Why is the `ELBSecurityPolicy` name hard-coded like this...? 🤔

Reply:
Mmm you mean exposing a variable to let the user select the policy? 🤔 

Btw, I noticed that we should be using `ELBSecurityPolicy-FS-2018-06` by default instead because this one implements Forward Secrecy method... (I'm not an expert on encryption stuff, but seems to be a good choice)